common/persistence/dataManagerInterfaces.go

```diff
@@ -964,6 +964,8 @@ type (
 		Encoding common.EncodingType // optional binary encoding type
 
 		DomainName string
+
+		TTLInSeconds int
```

Comment:
`UpdateWorkflowMutation` and `NewWorkflowSnapshot` in this struct both requires a TTL, and their TTLs should be different. What does this TTL belong to?

Reply:
Could you move the field to `WorkflowMutation` if it's only for update?